requirements.txt

```diff
@@ -0,0 +1,5 @@
+csp-billing-adapter
```

Comment:
This should updated to add a git repo source for the package since it isn't published to PyPI. See https://github.com/SUSE-Enceladus/csp-billing-adapter-k8s/blob/main/requirements.txt#L1